datadog.conf.example

```diff
@@ -47,7 +47,8 @@ api_key:
 # and dice per monitored app (= running Agent Check) on Datadog's backend.
 # create_dd_check_tags: no
 
-# Collect AWS EC2 custom tags as agent tags (requires an IAM role associated with the instance)
+# Collect AWS or OpenStack EC2 custom tags as agent tags
```

Comment:
No, it doesn't need to mention openstack here. I just thought it should be updated because now it impacts both cloud providers instead of just AWS. I'll update when I get into the office today.